pkpdapp/pkpdapp/models/dataset.py

```diff
@@ -17,9 +19,10 @@ class Dataset(models.Model):
     ]
     name = models.CharField(max_length=100, help_text='name of the dataset')
     datetime = models.DateTimeField(
+        default=timezone.now,
         help_text=(
-            'Date/time the experiment was conducted. ',
-            'All time measurements are relative to this date/time',
+            'Date/time the experiment was conducted. '
```

Comment:
Do we want this as a required field or optional? I would have thought that some of the time this wasn't really relevant if, for example, reusing old data.

Reply:
yea, probably should be optional, I'll change that